fe/fe-core/src/main/java/com/starrocks/lake/StarOSAgent.java

```diff
@@ -564,4 +565,29 @@ public boolean queryMetaGroupStable(long metaGroupId) {
         }
         return false; // return false if any error happens
     }
+
+    public List<Long> getWorkersByWorkerGroup(List<Long> workerGroupIds) throws UserException {
```

Comment:
if allow multiple groups to be listed together, return type should be `List<List<Long>>`, suggest only list single workgroup for now.